src/helpers.js

```diff
@@ -87,17 +87,47 @@ export function applySugar(environment: Object) {
 }
 
 export function getCommand(): { command: string, options: Object, parameters: Array<string> } {
+  // Print the environment separated by \0
+  const shScript = ('python -c "import os;print(\\"\\0\\".join(map(\\"=\\".join, dict(os.environ).items()))+\\"\\0\\")"||' +
```

Comment:
In all new ubuntu installations, `python3` is the default name, if we could add a case for that as well, that would be perfect